idseq_dag/steps/run_star.py

```diff
@@ -61,6 +61,12 @@ def run(self):
             command.execute(f"mv {src} {dst}")  # Move out of scratch dir
         command.execute("cd %s; rm -rf *" % scratch_dir)
 
+    def count_reads(self):
+        # count input (since run_star is the first step):
+        self.counts_dict = self.total_counts_from_star
```

Comment:
hm i am not sure about this~ this introduced some entanglement. I think we shouldn't do this at all but just count the raw fastq.gz up to 75M. (It's less optimized but I think the complexity introduced here is not worth it. )